src/main/resources/structure/root/main.gradle.mustache

```diff
@@ -94,3 +94,6 @@ tasks.withType(JavaCompile) {
             '-Amapstruct.suppressGeneratorTimestamp=true'
     ]
 }
+tasks.named('wrapper') {
+  gradleVersion = '6.7' 
```

Comment:
Have you consider change it for a centralized constant?

Reply:
solved